modules/apps/change-tracking/change-tracking-web/src/main/resources/META-INF/resources/publications/global_settings.jspf

```diff
@@ -61,4 +92,10 @@
 		'<portlet:namespace />unapprovedChanges',
 		<%= publicationsConfigurationDisplayContext.isPublicationsEnabled() ? false: null %>
 	);
+
+	Liferay.Util.toggleBoxes(
+		'<portlet:namespace />enablePublications',
+		'<portlet:namespace />manageRemotely',
+		<%= publicationsConfigurationDisplayContext.isPublicationsEnabled() ? false: null %>
```

Comment:
sf: missing space after `false`

Reply:
@pei-jung I originally had it as a boolean but then when it was disabled, I had no way to fetch or delete the OAuth2Application without the id to use for displaying the clientId and clientSecret and cleaning it up if disabled